src/main/java/org/simplify4u/plugins/PGPVerifyMojo.java

```diff
@@ -306,11 +314,13 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             final SkipFilter pluginFilter = preparePluginFilters();
             prepareForKeys();
 
-            final ArtifactResolver resolver = new ArtifactResolver(getLog(),
-                    repositorySystem, localRepository, remoteRepositories);
+            final ArtifactResolver resolver = new ArtifactResolver(getLog(), repositorySystem, localRepository,
+                    remoteRepositories);
             final Configuration config = new Configuration(dependencyFilter, pluginFilter, this.verifyPomFiles,
-                    this.verifyPlugins, this.verifyAtypical);
+                    this.verifyPlugins, this.verifyPluginDependencies, this.verifyAtypical);
             final Set<Artifact> artifacts = resolver.resolveProjectArtifacts(this.project, config);
+            getLog().info("Validating " + artifacts.size() + " artifacts ...");
+            getLog().debug("Discovered project artifacts: " + artifacts);
```

Comment:
please add ifDebugEnabled ... to not convert  artifacts set to String when not needed